### DISCUSSION TITLE
Add models in Discussions

### DISCUSSION THREAD

**Discussion by ghost** [community/community#16684] [Discussions] [answered]
Hi all!

#### 1. Feature name
Add models in Discussions

#### 2. Feature description
I would like to have a Discussions model like in the Issues or Pull Requests area on GitHub.

#### 3. Concept
![image](https://user-images.githubusercontent.com/95204402/168515270-4582c3e8-dd52-4bb4-a367-f0164df63407.png)

#### 4. Image description
As we can see in the image we have a model to do Discussions. The template is a markdown or text file that exemplifies a series of steps to be taken before starting a conversation or even a chat or even discussing an idea or feature.

#### 5. Why is this interesting?
1. This makes it possible for one or more open source communities to have clear and perhaps more efficient conversation processes - because the models allow a sequence to be done, a process - an approach.
2.  It allows people to have a better orientation in terms of discussion, idea.
3. This feature is common in the Issues and Pull Request area, I would find it interesting to have a Discussions model in order to have a model that everyone can use or reuse in the way they find most interesting.
4. Software like [Discourse](https://meta.discourse.org/) has this feature.
5. It would be really cool if GitHub had this feature, it would be a groundbreaking feature and maybe help most open source communities. The reason for this is that it saves time and is a guideline. Which allows people to follow a certain list of steps before Disputing.

#### 6. Notes
1. I hope to help the GitHub community in a positive way
2. I'm not promoting Discourse, rather I'm looking at a common feature in Discourse software and seeing if the community - people in general think it would be interesting to have this same feature on GitHub
3. I didn't find any reference here about the possibility of creating discussion templates so I requested it here
4. I would like to have positive or negative feedback on this idea

#### 7. References
- https://github.com/stevemao/github-issue-templates

**Comment by michael-hawker** [comment, marked as answer]
🦙 Duplicate of https://github.com/github/feedback/discussions/2838
